system/classes/Kohana/Date.php

```diff
@@ -279,9 +279,16 @@ public static function months($format = NULL)
 
 		if ($format === Date::MONTHS_LONG OR $format === Date::MONTHS_SHORT)
 		{
+			if ($format === Date::MONTHS_LONG) {
```

Comment:
Wouldn't it be better to change the Date::MONTHS_LONG and Date::MONTHS_SHORT values on lines 22-23? Or are you worried that this would introduce breaking changes?